backends/arm/test/models/test_mobilenet_v2_arm.py

```diff
@@ -97,3 +97,18 @@ def test_mv2_u55_BI(self):
             .partition()
             .to_executorch()
         )
+
+    def test_mv2_u85_BI(self):
```

Comment:
share with u55? And run it on corstone in the future?

Reply:
> share with u55? 

How do you mean? We already have a test_mv2_u55_BI() ?

> And run it on corstone in the future?

Maybe we are trying out different ways to run larger models like mv2 and others in a different way then the unit test setup. Lest see how it turns out :)

Reply:
Share the body of the test I mean, not a big deal TBH